custom_components/ramses_cc/__init__.py

```diff
@@ -7,16 +7,16 @@
 import logging
 from typing import Any
 
+from ramses_rf.entity_base import Entity as RamsesRFEntity
```

Comment:
This (**`Entity`**) isn't likely to be stable... that part of **ramses_rf** is a real mess. OK for now, but something to think about in future.

Reply:
OK, noted. It was this naming that suggested to me that rf_entity would make sense, but if it's subject to change anyway then leaving it as _device makes as much sense as anything else.